curation_portal/migrations/0005_prevent_blank_verdicts.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.4 on 2019-10-30 18:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("curation_portal", "0004_remove_blank_verdicts")]
+
+    operations = [
+        migrations.AlterField(
+            model_name="curationresult",
+            name="verdict",
+            field=models.CharField(max_length=25, null=True),
```

Comment:
I think you should explicitly set `blank=False` if you want to name it `prevent_blank_verdicts`